packages/react/src/primitives/PhoneNumberField/__tests__/PhoneNumberField.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { render, screen } from '@testing-library/react';
+
+import { PhoneNumberField } from '../PhoneNumberField';
+
+describe('PhoneNumberField primitive', () => {
+  it('should render a country code selector', async () => {
```

Comment:
Please add the following tests:
```
  it('should render classname for PhoneNumberField` async () => {
  it('should be able to set a size', async () => {
  it('should be able to set a variation', async () => {
  it('should fire onChange and onInput', async () => {
```
See:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/PasswordField/__tests__/PasswordField.test.tsx#L11